### ISSUE TITLE
Search auto-complete results not accessible to screen reader users

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#286]
# What’s the issue?

Screen reader users are unable to access the search suggestions.

For example, when JAWS is enable, screen reader users will hear the number of
search suggestions and instructions to use the arrow keys to navigate through the
suggestions. Using the down arrow does not allow screen reader users to navigate
through the search suggestions – it goes to the next item on the page.

The auto-complete also doesn’t use the recommended design pattern from the
WAI-ARIA Authoring Practices 1.1.

# Why it matters

Although screen reader users are still able to complete a free-text search, they
cannot benefit from  the search suggestions functionality.

# Our recommendation

Ensure that screen reader users, including desktop and mobile can navigate through
search suggestions using arrow keys on keyboard and swipe on mobile.
See accessible auto-suggest from WebAxe and the Combobox ARIA design pattern.
